pylint/config/option_manager_mixin.py

```diff
@@ -259,8 +259,8 @@ def read_config_file(
             if config_file.suffix == ".toml":
                 try:
                     self._parse_toml(config_file, parser)
-                except tomllib.TOMLDecodeError as e:
-                    self.add_message("config-parse-error", line=0, args=str(e))  # type: ignore[attr-defined]
+                except tomllib.TOMLDecodeError:
+                    pass
```

Comment:
This is now raised by the new parser and would lead to double messages otherwise.